### PR TITLE
AAE-17010 Upgrade Java Content SDK to Spring Boot 3.2

### PR DESCRIPTION
**Pull request by igdianov**
https://alfresco.atlassian.net/browse/AAE-17010

### CODE REVIEW

File: alfresco-acs-java-rest-api/alfresco-acs-java-rest-api-lib/src/main/resources/templates/libraries/spring-cloud/pom.mustache [hunk 1]
```diff
@@ -10,7 +10,7 @@
   </parent>
   <artifactId>{{artifactId}}</artifactId>
   <name>{{artifactId}}</name>
-  <version>{{artifactVersion}}</version>
+  <version>@project.version@</version>
```
Comment (matteo4diani) [inline alfresco-acs-java-rest-api/alfresco-acs-java-rest-api-lib/src/main/resources/templates/libraries/spring-cloud/pom.mustache:13]: @igdianov what is this syntax? I cannot find relevant examples online :)

**Review by matteo4diani — APPROVED** [review]
LGTM 🚀

File: alfresco-acs-java-rest-api/alfresco-acs-java-rest-api-lib/src/main/resources/templates/libraries/spring-cloud/pom.mustache (hunk 1 above)
Comment (graric) [reply alfresco-acs-java-rest-api/alfresco-acs-java-rest-api-lib/src/main/resources/templates/libraries/spring-cloud/pom.mustache:13]: @matteo4diani [Take a look here](https://docs.spring.io/spring-boot/docs/1.5.6.RELEASE/reference/html/howto-properties-and-configuration.html#howto-automatic-expansion-maven)

**Review by graric — APPROVED** [review]

File: alfresco-acs-java-rest-api/alfresco-acs-java-rest-api-lib/src/main/resources/templates/libraries/spring-cloud/pom.mustache (hunk 1 above)
Comment (matteo4diani) [reply alfresco-acs-java-rest-api/alfresco-acs-java-rest-api-lib/src/main/resources/templates/libraries/spring-cloud/pom.mustache:13]: Thanks @graric

**Pull request merged**